### PR TITLE
ext-costs moved to logic config extracted

### PR DESCRIPTION
**Pull request by fckt**
The goal is to remove duplication of RuntimeFees in `wasm_config` and in `runtime_config` (see testnet.json changes). Would be glad to get ideas on how to make it better..

### CODE REVIEW

File: runtime/near-vm-logic/tests/test_registers.rs [hunk 1]
```diff
@@ -85,8 +91,9 @@ fn test_max_register_memory_limit() {
     let context = get_context(vec![], false);
     let config = Config::free();
     let promise_results = vec![];
+    let fees = RuntimeFeesConfig::default();
```
Comment (nearmax) [inline runtime/near-vm-logic/tests/test_registers.rs]: Should we have `RuntimeFeesConfig::free()` here?

File: runtime/near-vm-logic/src/config.rs [hunk 2]
```diff
@@ -1,13 +1,12 @@
 use crate::types::Gas;
-use near_runtime_fees::RuntimeFeesConfig;
 use serde::{Deserialize, Serialize};
 use std::collections::hash_map::DefaultHasher;
 use std::hash::{Hash, Hasher};
 
 #[derive(Clone, Debug, Hash, Serialize, Deserialize)]
 pub struct Config {
```
[the remark this answers is not in the document]
Comment (fckt) [reply runtime/near-vm-logic/src/config.rs]: VMConfig would be better I think.

**Comment by fckt** [comment]
`VMLogicBuilder` was introduced as well to reduce setup boilerplate in tests.

File: chain/epoch_manager/src/lib.rs [hunk 3]
```diff
@@ -992,17 +994,7 @@ mod tests {
         epoch_manager
             .record_block_info(
                 &h[1],
```
Comment (nearmax) [inline chain/epoch_manager/src/lib.rs]: Could you check whether you have https://github.com/nearprotocol/nearcore/blob/master/rustfmt.toml in ~/.config/rustfmt/ ?

Comment (fckt) [reply chain/epoch_manager/src/lib.rs]: Nope, I don't have `~/.config/rustfmt/ `. I've started to use Idea, maybe that's why..

Comment (nearmax) [reply chain/epoch_manager/src/lib.rs]: FYI `~/.config/rustfmt/` is independent on IDE, it is just where rustfmt looks for config.

Comment (fckt) [reply chain/epoch_manager/src/lib.rs]: of course. But who knows, maybe idea does more magic. But I checked it, rust fmt really wants to format it! You can check it by running `cargo fmt check` on staging branch.

**Pull request merged**